src/osd/OSDCap.cc

```diff
@@ -85,10 +85,6 @@ ostream& operator<<(ostream &out, const OSDCapPoolTag &pt)
 
 ostream& operator<<(ostream& out, const OSDCapMatch& m)
 {
-  if (m.auid != -1LL) {
-    out << "auid " << m.auid << " ";
-  }
-
```

Comment:
The code changes all look fine, but this commit message includes a separate line "fix osdcap" after the s-o-b statement?